packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/colorPalette.ts

```diff
@@ -56,3 +56,38 @@ export const ColorBlindPalette: FlamegraphPalette = {
   neutralColor: Color.rgb(148, 142, 142),
   badColor: Color.rgb(220, 50, 32),
 };
+
+export const TimelineSeriesPalette: FlamegraphPalette = {
```

Comment:
I think this is just a poc (based on the pr title), and you won't merge this yet

but I think it's important to say that I don't think this should belong here. The flamegraph component itself has nothing to do with the timeline.  So I recommend moving this to as closest as possible to where it's used (TagExplorer component most likely).

Reply:
yeah I was going to put it here but I do like these colors better for the timeline so I'll move it closer to there